botorch/acquisition/input_constructors.py

```diff
@@ -1239,3 +1241,39 @@ def optimize_objective(
         return_best_only=True,
         sequential=sequential,
     )
+
+
+# TODO make single-objective with pairwise and multi-objective with pareto
```

Comment:
Is this something you plan to do, or something you think would be good for someone else to do? (If the latter, a GH issue would be better than a comment)

Reply:
Hmm. I actually cannot recall putting that comment in (but I probably did). 

Otherwise, but it could refer to this:
https://github.com/pytorch/botorch/issues/1763